content/tutorials/use-simple-xrp-payments/send-xrp.md

```diff
@@ -0,0 +1,593 @@
+
+```js
+api.on('ledger', ledger => {
+  console.log("Ledger version", ledger.ledgerVersion, "was just validated.")
+  if (ledger.ledgerVersion > maxLedgerVersion) {
+    console.log("If the transaction hasn't succeeded by now, it's expired")
+  }
+})
+```
+
+<div class="interactive-block" id="interactive-wait">
+<div class="interactive-block-inner">
+  <div class="breadcrumbs-wrap">
+    <ul class="breadcrumb prereqs">
+      <li class="breadcrumb-item active bc-generate"><a href="#interactive-generate">Generate</a></li>
+      <li class="breadcrumb-item disabled bc-connect"><a href="#interactive-connect">Connect</a></li>
+      <li class="breadcrumb-item disabled bc-prepare"><a href="#interactive-prepare">Prepare</a></li>
+      <li class="breadcrumb-item disabled bc-sign"><a href="#interactive-sign">Sign</a></li>
+      <li class="breadcrumb-item disabled bc-submit"><a href="#interactive-submit">Submit</a></li>
+      <li class="breadcrumb-item disabled current bc-wait"><a href="#interactive-wait">Wait</a></li>
+      <li class="breadcrumb-item disabled bc-check"><a href="#interactive-check">Check</a></li>
+    </ul><!--/.breadcrumb.prereqs-->
+  </div><!--/.breadcrumbs-wrap-->
+  <table>
+    <tr>
+      <th>Latest Validated Ledger Version:</th>
+      <td id="current-ledger-version">(Not connected)</td>
+    </tr>
+      <tr>
+        <th>Ledger Version at Time of Submission:</th>
+        <td id="earliest-ledger-version">(Not submitted)</td>
+      </tr>
+    <tr>
+      <th>Transaction LastLedgerSequence:</th>
+      <td id="tx-lls"></td>
+    </tr>
+  </table>
+</div>
+</div>
+<script type="application/javascript">
+api.on('ledger', ledger => {
+  $("#current-ledger-version").text(ledger.ledgerVersion)
+
+  if ( $(".breadcrumb-item.bc-wait").hasClass("active") ) {
+    // Advance to "Check" as soon as we see a ledger close
+    $(".breadcrumb-item.bc-wait").removeClass("active")
+    $(".breadcrumb-item.bc-wait").addClass("done")
+    $(".breadcrumb-item.bc-check").removeClass("disabled")
+    $(".breadcrumb-item.bc-check").addClass("active")
+    $("#get-tx-button").prop("disabled", false)
+  }
+})
+</script>
+
+
+### {{n.next()}}. Check Transaction Status
+
+To know for sure what a transaction did, you must look up the outcome of the transaction when it appears in a validated ledger version. For example, you can use the [getTransaction() method](rippleapi-reference.html#gettransaction) to check the status of a transaction:
+
+```js
+// Continues from previous examples.
+// earliestLedgerVersion was noted when the transaction was submitted.
+// txID was noted when the transaction was signed.
+tx = await api.getTransaction(txID, {minLedgerVersion: earliestLedgerVersion})
+console.log("Transaction result:", tx.outcome.result)
+console.log("Balance changes:", JSON.stringify(tx.outcome.balanceChanges))
+```
+
+The RippleAPI `getTransaction()` method only returns success if the transaction is in a validated ledger version.
```

Comment:
It will raise an exception, I think. Added a try/catch to the code sample and the actual code.

But there's also a question for @intelliot about whether providing a minLedgerVersion may cause it to search non-validated ledgers. (I think there may be a case where it returns an error because it fails to find the `ledger_index` field in `rippled`'s response while doing so.)